sklego/meta/estimator_transformer.py

```diff
@@ -38,6 +39,34 @@ def transform(self, X):
         Returns array of shape `(X.shape[0], )` if estimator is not multi output.
         For multi output estimators an array of shape `(X.shape[0], y.shape[1])` is returned.
         """
-        check_is_fitted(self, "estimator_")
+        check_is_fitted(self)
         output = getattr(self.estimator_, self.predict_func)(X)
         return output if self.multi_output_ else output.reshape(-1, 1)
+
+    def get_feature_names_out(self, feature_names_out=None) -> list:
+        """
+        Defines descriptive names for each output of the (fitted) estimator.
+        :param feature_names_out: Redundant parameter for which the contents are ignored in this function.
```

Comment:
Have you checked if this renders nicely in sphinx? I wouldn't mind some newlines in the docstring.